### PR TITLE
Adding reverse rates to CNO_extras

### PR DESCRIPTION
**Pull request by simonguichandut**
(and re-compiling the network)

### CODE REVIEW

**Comment by zingale** [comment]
I think that your version of pynucastro is a little out of date -- we switched from using `std::pow()` to using `amrex::Math::powi<>()` for powers, for better GPU performance.  I don't think we've made a new pynucastro release since then, so you'll need to update your pynucastro git.

**Comment by zingale** [comment]
for some reason, there are conflicts, so I think that you need to update your `development` branch and merge it into this branch.  That will make git grumpy, so you may just want to rerun the script afterwards to make everything happy.

**Comment by simonguichandut** [comment]
Ok thanks, I updated it.  I still see some uses of `std::pow()`. From grep:
> ./table_rates.H:    rate       = std::pow(10.0_rt, entries(jtab_rate));
./table_rates.H:    edot_nu    = -std::pow(10.0_rt, entries(jtab_nuloss));
./table_rates.H:    edot_gamma = std::pow(10.0_rt, entries(jtab_gamma));
./partition_functions.H:            pf = std::pow(10.0_rt, log10_pf);

They're gone from `actual_rhs.H` though.

**Comment by zingale** [comment]
okay, it seems we didn't get the table versions done.  I'll file a separate issue for that.

I still see conflicts though.

**Comment by zingale** [comment]
oh, wait... those std::pow() are fine.  It is only integer powers that can be replaced.

File: networks/CNO_extras/actual_rhs.H [hunk 1]
```diff
@@ -838,11 +868,19 @@ void rhs_nuc(const burn_t& state,
         (screened_rates(k_p_F18_to_Ne19)*Y(F18)*Y(H1)*state.rho + -screened_rates(k_Ne19_to_p_F18)*Y(Ne19));
 
     ydot_nuc(Ne20) =
+<<<<<<< HEAD
         (screened_rates(k_He4_O16_to_Ne20)*Y(He4)*Y(O16)*state.rho + -screened_rates(k_Ne20_to_He4_O16)*Y(Ne20)) +
```
Comment (zingale) [inline networks/CNO_extras/actual_rhs.H:872]: merge conflicts

Comment (simonguichandut) [reply networks/CNO_extras/actual_rhs.H:872]: whoops

**Review by zingale — APPROVED** [review]

**Pull request merged**